variables.tf

```diff
@@ -27,6 +27,12 @@ variable "cluster_security_group_id" {
   default     = ""
 }
 
+variable "cluster_additional_security_group_ids" {
+  description = "A list of additional security group ids to attach to EKS cluster"
```

Comment:
this should be exactly same as in `README.md` as that file is generated by `terraform-docs` basing on variable/output descriptions